awx/api/serializers.py

```diff
@@ -2768,6 +2768,20 @@ def create(self, validated_data):
                 value = validated_data.pop(attr)
                 if value:
                     validated_data['inputs'][attr] = value
+
+        # XXX - Temporary hack for dev and demonstration purposes
```

Comment:
Before we merge, let's definitely replace this with new API views+serializers that let you manage `CredentialInputSource` objects.

Reply:
agree - sounds like a plan!